connectors/bitbucket-server/src/main/resources/static/discovery/metadata.json

```diff
@@ -1,11 +1,23 @@
 {
   "authorization_header": "x-bitbucket-server-authorization",
   "backend_base_url_header": "x-bitbucket-server-base-url",
```

Comment:
Once I see that the new format is being used, I'm not going to read these values. I expect "new" connectors to look for 'X-Connector-Authorization` and `X-Connector-Base-Url`.

Reply:
Cool, I'll add changes to this branch itself.

Reply:
Connectors now look for standard set of headers.